Python/pywarpx/_libwarpx.py

```diff
@@ -366,7 +376,7 @@ def getCellSize(direction, level=0):
 #
 #    libwarpx.warpx_ComputePMLFactors(lev, dt)
 
-def add_particles(species_number=0,
+def add_particles(species_name,
```

Comment:
Was being able to identify particles via `species_name` added recently, or was it always there, but the Python side was never updated?

Reply:
Yeah it seems that way.